record-and-playback/core/scripts/archive/archive.rb

```diff
@@ -111,6 +111,37 @@ def delete_audio(meeting_id, audio_dir)
   end
 end
 
+def remux_and_archive(source_dir, dest_dir)
+  files = Dir.glob("#{source_dir}/*")
+
+  if files.empty?
+    BigBlueButton.logger.warn("No media files found in #{source_dir}")
+    return
+  end
+
+  BigBlueButton.logger.info("Remuxing and archiving files at #{source_dir}")
+  FileUtils.mkdir_p(dest_dir)
+
+  files .each do |file|
+    ext = File.extname(file)
+    next if ext.empty?
+    # These can potentially be webm (VP8/VP9), mp4 (H.264), or mkv (VP8/VP9/H.264)
+    output_basename = File.join(dest_dir, File.basename(file, ext))
+    format = {
+      extension: ext.delete('.'),
+      parameters: [
+        %w[-c copy],
+      ]
+    }
+    BigBlueButton::EDL.encode(
+      nil,
+      file,
+      format,
+      output_basename
+    )
+  end
```

Comment:
My only concern here is that during incremental archiving (segmented recordings), this will truncate the destination file (which might be in use!) and rewrite it. It would be better for the remux to write to a file with a temporary file name, and then rename it to the final filename once the remux is complete.

Reply:
Ah, I'd forgotten about that. Yeah, should be good to go as-is!